docs/user/instrument-android-crashes-example.md

```diff
@@ -0,0 +1,198 @@
+# Recording Android Crashes with Glean
```

Comment:
This uses the word "instrumenting" rather than "recording" every where this is linked to.  I guess I prefer "instrumenting", but don't have a strong preference.  Let's make it consistent, though?